extras/Build/CMake/JUCEUtils.cmake

```diff
@@ -1337,6 +1347,40 @@ function(_juce_set_plugin_target_properties shared_code_target kind)
         endif()
 
         _juce_copy_after_build(${shared_code_target} ${target_name} "${output_path}" JUCE_VST3_COPY_DIR)
+    elseif(kind STREQUAL "LV2")
+        set_target_properties(${target_name} PROPERTIES
+            BUNDLE_EXTENSION lv2
+            PREFIX ""
+            SUFFIX .lv2
+            BUNDLE TRUE
+            XCODE_ATTRIBUTE_WRAPPER_EXTENSION lv2
+            XCODE_ATTRIBUTE_LIBRARY_STYLE Bundle
+            XCODE_ATTRIBUTE_GENERATE_PKGINFO_FILE YES)
+
+        _juce_create_windows_package(${shared_code_target} ${target_name} lv2 "" x86-win x86_64-win)
+
+        set(output_path "${products_folder}/${product_name}.lv2")
+
+        if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+            # On linux we assume that the output arch is the same as the that of the host platform
+            set(is_platform_x64 $<EQUAL:${CMAKE_SIZEOF_VOID_P},8>)
+            set(arch_string $<IF:${is_platform_x64},x86_64,i386>)
+
+            set_target_properties(${target_name} PROPERTIES
+                SUFFIX .so
+                LIBRARY_OUTPUT_DIRECTORY "${output_path}")
+        endif()
```

Comment:
Correct me if I'm wrong, but I don't think we need any of this.

This would put the output files in `<build_folder>/<plugin_name>_artifacts/LV2/<plugin_name>.lv2`, whereas I think we could just as easily be okay with `<build_folder>/<plugin_name>_artifacts/LV2`.

And I don't think the `if(CMAKE_SYSTEM_NAME STREQUAL "Linux")` does anything. The output suffix should already be `.so`, and I don't think the `is_platform_x64` and `arch_string` variables are being used for anything.

Reply:
these lines (and some above) are the my blind-copy of VST3 macros, so, they may be completely unnecessary